dialogue/Tutorial/TutQuantumHeart.json

```diff
@@ -20,8 +20,8 @@
       "Przedmiot: Serce Kwantowe.\nOpracowane przez: Niesamowite & Prawdopodobne do Rozdarcia Czasoprzestrzeni Innowacje.",
       "Użycie: Wiąże użytkownika do równoległego podmiotu istniejącego w innym wymiarze rzeczywistości.",
       "To kwantowe powiązanie spowoduje, że wiązany podmiot naśladuje ruchy użytkownika, i odwrotnie.",
-      "Powiązanie może być rozdarte uzywając umiejętności, spowodowanie kwantowe preniesienie użytkownika do lokalizaji wiązanego podmiota.",
-      "<s>UŻYWAJ OSTROŻNIE!!</s>\nWszyscy istniejemy w tej samej czasoprzestrzeni, więc proszimy nie rozedrzeć ją, m'kej?"
+      "Powiązanie może być rozdarte używając umiejętności, spowodowanie kwantowe przeniesienie użytkownika do lokalizacji wiązanego podmiota.",
+      "<s>UŻYWAĆ OSTROŻNIE!!</s>\nWszyscy istniejemy w tej samej czasoprzestrzeni, więc prosimy nie rozedrzeć ją, m'kej?"
```

Comment:
"prosimy jej nie rozdzierać"
Also I think it would be better to just use "okej?", because m'kej doesn't really work in polish